party3.tex

```diff
@@ -0,0 +1,33 @@
+
+
+Though the dev guide has a procedure \footnote{\url{https://developer.lsst.io/stack/packaging-third-party-eups-dependencies.html}}
+for making a third part package for inclusion in the pipelines distrib we have no official policy for how we accept and deal with such packages.
+
+
+
+There are several reasons to include third part code in our distributions:
+\begin{enumerate}
+\item There are packages which we use in our code e.g. starlink\_ast.  These are standard dependencies.  Currently these are put in lsst github org and probably should not be. Apart from that this is a well understood reason for third party code. \label{item:depend}
+\item There are packages which we want to use when working with the deployed software. AstroPy might be such an example - we don't need it to build or run our code but its sure handy for interaction with the data.\label{item:want}
+\item A collaborator has a package which depends on our code and is intended for some form of post processing. Another example here would be an algorithm we want to use such as ngmix which we do not necessarily need to support. This is what most people on LSST are thinking about under the topic of third party software. \label{item:colab}
```

Comment:
This actually seems like two items:

- A collaborator has some code which *they* think is useful, and want to use to (post-) process LSST data;
- A collaborator has some code which *we* think is useful, and would like to incorporate into LSST processing (ngmix falls into this latter category).

I don't think these should be regarded as the same.